src/freeclimb.Test/Api/DefaultApiTests.cs

```diff
@@ -123,6 +123,23 @@ public void CreateAnApplicationTest()
             Assert.IsAssignableFrom<ApplicationResult>(response);
         }
         
+        /// <summary>
+        /// Test CreateKnowledgeBaseCompletion
+        /// </summary>
+        [Fact]
+
+        public void CreateKnowledgeBaseCompletionTest()
+        {
+            
+            string knowledgeBaseId = knowledgeBaseIdTestValue();
+	        
+            CompletionRequest completionRequest = completionRequestTestValue();
+	        
+            var response = instance.CreateKnowledgeBaseCompletion(knowledgeBaseId, completionRequest);
+            //Assert.IsType<CompletionResult>(response);
```

Comment:
Why is this commented out?

Reply:
I believe this assertion is not needed, will go ahead and remove the assertion